examples/fabric-gateway/TEST.md

```diff
@@ -0,0 +1,155 @@
+# Terraform examples for Fabric Gateway (FG)
+
+## Create FG
+
+```cd fabric-gateway/
```

Comment:
Are there manual steps that happen outside of terraform for these tests (for example, a person has to respond to a ticket or click a button in order to complete the provisioning process for a resource)?  If it's all terraform, it would be preferable to create a test case or cases like the others in the `tests/` directory.

Reply:
This was just a helper shell script(Now modified into a .md file) we created for the demo in addition to actual terraform tests under `/tests` directory.

Reply:
I see that this markdown file was created in response to an earlier comment on the shell script.  Given the current state of end-to-end testing for these resources, I think we can merge this doc as-is.